inc/model.class.php

```diff
@@ -828,30 +826,30 @@ public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $
         if ($item->getType() == __CLASS__) {
```

Comment:
To test, if it's ok, you should be able to remove a few more phpstan-ignore:
```suggestion
        if ($item->getType() == __CLASS__ && $item instanceof self) {
```